core/chains/evm/types/models.go

```diff
@@ -76,6 +75,11 @@ func (h *Head) GetParent() commontypes.Head[common.Hash] {
 	return h.Parent
 }
 
+// Return 0 since block difficulty is not relevant for EVM Chains.
```

Comment:
Remove comment. We don't know how every EVM chain works, and there are always exceptions. For all we know some other chain may be utilizing difficulty so here we would add a special case by introducing a new `chainType`.